### ISSUE TITLE
Question regarding connection disconnect/close.

### ISSUE THREAD

**Issue by jmargieh** [amqp/rhea-promise#90]
Hello all,

In cases the connection was closed/disconnected.
If the connection was created with ```reconnect: true```. does rhea handles the underlying Links/Sessions ?

Thanks.

**Comment by grs** [comment]
Yes, rhea will re-establish sessions and links that were created before the reconnect.

**Comment by jmargieh** [comment]
@grs thank you for your quick response. will they be re-established with the same exact same senders/receivers configurations?

**Comment by grs** [comment]
yes
